dotcom-rendering/cypress/e2e/parallel-7/snapshot-standard-layout.e2e.cy.js

```diff
@@ -0,0 +1,23 @@
+import { disableCMP } from '../../lib/disableCMP.js';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+import { breakpoints } from '@guardian/source-foundations';
+import '@percy/cypress';
+
+describe('Snapshot standard layout', function () {
+	beforeEach(function () {
+		disableCMP();
```

Comment:
Not for this PR, but perhaps we could have a test that allows the banner to show, dismisses it, refreshes the page, and verifies a different banner shows. We already have tests that check this in other places but given how critical this content is an extra level of assurance would be good